cmd/res/configuration.toml

```diff
@@ -0,0 +1,83 @@
+[Writable]
+LogLevel = "INFO"
+  # InsecureSecrets are required for when Redis is used for message bus
+  [Writable.InsecureSecrets]
+    [Writable.InsecureSecrets.DB]
+    path = "redisdb"
+      [Writable.InsecureSecrets.DB.Secrets]
+      username = ""
+      password = ""
+
+[Service]
+HealthCheckInterval = "10s"
+Host = "localhost"
+ServerBindAddr = ""  # blank value defaults to Service.Host value
+Port = 49995
```

Comment:
We have moved all device services into the 599** range to comply with industry standards

Reply:
This issue has been fixed in this commit b41907f: change port to 59911

Reply:
> We have moved all device services into the 599** range to comply with industry standards

This issue has been fixed in this commit b41907f: change port to 59911